sdk/js/src/Client.ts

```diff
@@ -54,8 +55,17 @@ export class DVCClient implements Client {
 
         const storedAnonymousId = this.store.loadAnonUserId()
         this.user = new DVCPopulatedUser(user, options, undefined, storedAnonymousId ?? undefined)
-
         this.options = options
+        if (!this.options.configCacheTTL) {
+            this.options.configCacheTTL = 604800000
+        }
+        
+        if (!this.options.disableConfigCache) {
+            this.getConfigCache()
```

Comment:
Are the variable initialize callbacks going to be called if we update the config cache?

Reply:
I'm not too sure, how can I check that?

Reply:
added a test case to check if the `variable.value` gets the same value as the one in config cache

Reply:
In the example app you should be able to define a client.variable.onUpdate callback outside of the client's onInitialized callback, and the onUpdate callback should be triggered with the value from the cache before the config is fetched. You may need to throttle the network connection to slow down the config fetch, or you can temporarily comment out the config fetch.

You'll likely need to look into what logic is triggered on a successful config fetch and call that from here as well.